DigitalLearningSolutions.Web/Views/Home/Welcome.cshtml

```diff
@@ -0,0 +1,35 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.Home
+@model LandingPageViewModel
+
+@{
+  ViewData["Title"] = "Welcome";
+}
+
+<link rel="stylesheet" href="@Url.Content("~/css/home/welcome.css")" asp-append-version="true"/>
+
+<div class="nhsuk-grid-row">
+  <div id="column" class="nhsuk-grid-column-full">
+    <h1 class="nhsuk-heading-xl">A platform for digital learning delivery</h1>
+
+    <p class="nhsuk-u-font-size-32">Enabling health and care organisations to deliver digital learning to staff and students.</p>
+
+    @if (Model.UserIsLoggedIn) {
+      <a class="nhsuk-button auth-button auth-button--left" href="@Model.CurrentSiteBaseUrl/Home?action=appselect">
+        Switch&nbsp;application

```

Comment:
I think we should handle it just like "Log in" and other buttons, using a normal space, unless the NHS buttons automatically tries to have a new line here.